docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,122 @@
+
+Note: There could be many tests within a project. Each test is checking a table created by your choice of input Models, data sources, and transformation Gems.
+
+By default, and most often, the condition for a project test is simply a check that the given query does not return any failing rows.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/mvk1axfn7e?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Set up a test
+
+Since project tests share the same canvas as Models, you can set them up the same way you would develop and run a Pipeline.
+
+### Develop a test
+
+To develop a project test, start by opening a project:
+
+1. Under the Project Browser, click **Add Test**.
+
+   ![Add a new test](img/project-test-add-new.png)
+
+2. Enter your **Test Name**. Project tests are saved to a tests SQL file by default.
+
+3. Click **OK** to create your new test.
+
+4. Create your test Model by dragging and dropping Gems to your Visual canvas. Connect them to your Data Test. You can also write your test on the Code view. You can use as many Models and other database objects as you’d like in your test.
+
+### Run a test
+
+After you’ve developed your project test and executed all of your Models, you can run the test.
+
+1. Click the **Play** button on either the canvas or an individual Gem to execute your test. The outputted table from the Gem is what’s tested.
+   1. Run a whole test. Clicking the canvas **Play** button executes the complete test and shows the interim data after the Data Test Gem. Additionally, it displays the final test status, succeeded, warning, or failed, in a summary.
+   2. Run a partial test. Clicking the Gem **Play** button executes the test SQL only for a particular Gem, and displays that Gem’s data on the output. This option doesn’t execute the data test.
+2. Click **See Run Details** to view the test Summary. Depending on the outcome of the test, the icon displays a different color.
+
+   ![See Run Details](img/project-test-run-details.png)
+
+   You can click to expand the test logs in the Summary to view the dbt logs.
+
+   ![View test summary](img/project-test-summary.png)
+
+In addition to the previous relational integrity test, you can create tests to check that the total payment amount from customers is positive or simply check that all of your tables are still accessible.
+
```

Comment:
Suggest adding: "Any series of transformation gems can be tested, because a Project Test is simply asserting that the resulting table meets a certain criteria."

Reply:
Added, but reworded a bit.